workflow-templates/im-build-dotnet-ci.yml

```diff
@@ -1,9 +1,13 @@
-# Workflow Code: LoathsomeSnipe_v10    DO NOT REMOVE
+# Workflow Code: LoathsomeSnipe_v11    DO NOT REMOVE
 
 # TODO: Ensure each of the repo-level and env-level secrets used in this workflow have been populated by an admin in your repository.
 
 # TODO:  If the project contains a local nuget.config remove it.  It interferes with restoring packages on the GitHub Action runners.
 
+# TODO:  If your project contains a .npmrc file that points to any of our im-* orgs as a registry you have two options:
+#        1 - Remove the .npmrc file completely and ensure you are using the im-open/authenticate-with-gh-package-registries action in each job that needs to authenticate
+#        2 - Supply a READ_PKG_TOKEN environment variable to each step that might use the .npmrc file for authentication with GH Packages (dotnet build, setup node, npm ci, etc)
```

Comment:
They could also specify it at the job or even workflow level, right? Do we for some reason want them to do it on the individual steps?

Reply:
It does make more sense at the job or workflow particularly if it spans steps, like build and test.

Reply:
I'm not going to lie my mind is kind of blown right now.  For some reason I thought if you wanted to an action to have access to an env variable, (like inside the action, not in the expressions setting up the action) you had to specifically pass it to the action that way.  I thought when we were very first starting out I didn't specify it that way and the action wasn't getting the value, but there might have been something else going on.